sirepo/pkcli/raydata.py

```diff
@@ -4,16 +4,48 @@
 :copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
-from __future__ import absolute_import, division, print_function
-from pykern import pkio
-from pykern import pksubprocess
 from pykern.pkcollections import PKDict
 from pykern.pkdebug import pkdp, pkdlog
 from sirepo.template import template_common
-import sys
 
 
+def run(cfg_dir):
+    _run()
+    template_common.write_sequential_result(PKDict())
+
 def run_background(cfg_dir):
-    pksubprocess.check_call_with_signals(
-        [sys.executable, template_common.PARAMETERS_PYTHON_FILE],
-    )
+    _run()
+
+
+def create_scans(num_scans, delay=True):
+    from sirepo.template import raydata
+    import bluesky
+    import bluesky.plans
+    import databroker
+    import ophyd.sim
+    import time
+
+    num_scans = int(num_scans)
+    assert num_scans > 0, \
+        f'num_scans={num_scans} must be > 0'
+    d = databroker.Broker.named(raydata.catalog().name)
+    RE = bluesky.RunEngine({})
+    RE.subscribe(d.insert)
+    b = set(raydata.catalog())
+    for i in range(num_scans):
+        RE(bluesky.plans.count(
+            [ophyd.sim.det1, ophyd.sim.det2],
+            num=5,
+            md={
+                'T_sample_': i,
+                'owner': 'foo',
+                'sequence id': i,
+            },
+        ))
+        a = set(raydata.catalog())
```

Comment:
Why is `a` needed?

Reply:
9abb7c424 not needed. leftover from testing